cli/templatecreate.go

```diff
@@ -357,3 +341,52 @@ func ParseProvisionerTags(rawTags []string) (map[string]string, error) {
 	}
 	return tags, nil
 }
+
+type checkTemplateCreateEntitlementsArgs struct {
+	client               *codersdk.Client
+	requireActiveVersion bool
+	defaultTTL           time.Duration
+	failureTTL           time.Duration
+	dormancyThreshold    time.Duration
+	dormancyAutoDeletion time.Duration
+	maxTTL               time.Duration
+}
+
+func checkTemplateCreateEntitlements(ctx context.Context, args checkTemplateCreateEntitlementsArgs) error {
+	isTemplateSchedulingOptionsSet := args.failureTTL != 0 || args.dormancyThreshold != 0 || args.dormancyAutoDeletion != 0 || args.maxTTL != 0
+
+	if isTemplateSchedulingOptionsSet || args.requireActiveVersion {
+		if args.failureTTL != 0 || args.dormancyThreshold != 0 || args.dormancyAutoDeletion != 0 {
+			// This call can be removed when workspace_actions is no longer experimental
+			experiments, exErr := args.client.Experiments(ctx)
+			if exErr != nil {
+				return xerrors.Errorf("get experiments: %w", exErr)
+			}
+
+			if !experiments.Enabled(codersdk.ExperimentWorkspaceActions) {
```

Comment:
Aside: I've noticed us call this "WorkspaceActions" and "WorkspaceCleanup" in different places. E.g. [here](https://www.notion.so/coderhq/Internal-Roadmap-b0b620f760bf4e3195a001764e0bad55?p=c278573c45634b13ae7e854365516b01&pm=s). @sreya can we standardize on one? Workspace Cleanup makes more sense to me.